rep-2010.rst

```diff
@@ -220,7 +219,34 @@ And the definitions for ``example_interfaces/srv/AddTwoInts_Reponse`` is,
 Configuration
 -------------
 
-TODO: Describe what configuration options there are and how we plan to expose them (e.g. with ROS parameters and/or environment variables).
+Configuration of service introspection features will be done through ROS parameters.
+
+.. code-block:: yaml
+
+   /NODE_NAME:
+      ros__parameters:
+         
+         # Indicates if CLIENT_SENT and CLIENT_RECEIVED events will be published
+         SERVICE_NAME/client_events_enabled: bool
+
+         # Indicates if SERVICE_SENT and SERVICE_RECEIVED events will be published
+         SERVICE_NAME/service_events_enabled: bool
+         
+         # Indicates if request content will be included in CLIENT_SENT RequestEvent messages
+         SERVICE_NAME/_enable_client_sent_content: bool
+
+         # Indicates if request content will be included in SERVICE_RECEIVED RequestEvent messages
+         SERVICE_NAME/_enable_service_receive_content: bool
+
+         # Indicates if response content will be included in SERVICE_SENT RequestEvent messages
+         SERVICE_NAME/_enable_service_sent_content: bool
+
+         # Indicates if response content will be included in CLIENT_RECEIVED RequestEvent messages
+         SERVICE_NAME/_enable_client_receive_content: bool
```

Comment:
Out of curiosity why aren't we going with the enum approach for `_enable_content`. I remember their were some reasons for not using enum's I guess but not sure. 

We could do something like:
`SERVICE_NAME/_enable_content corresponding_enum`

Reply:
There would be 2^4 possible configurations which doesn't sound ideal. This is also a ROS parameter and we would want to settable via YAML files

Reply:
I think the main problem is that it's difficult to reference enums when setting parameters from the CLI or YAML file. There is not "enum" type for ROS parameters.

Reply:
I've added a note about this to the rationale section [6419e51](https://github.com/jacobperron/rep/pull/2/commits/6419e51e61766535b7b0c5a62a960850d6fcabd2)